deploy/crds/devconsole_v1alpha1_gitsource_crd.yaml

```diff
@@ -9,6 +9,8 @@ spec:
     listKind: GitSourceList
     plural: gitsources
     singular: gitsource
+    shortNames:
+      - src
```

Comment:
`src` is too overloaded a term. How about gitsrc or gs ( along the lines of `bc` , `dc`)?

Reply:
I agree